cirq/circuits/circuit.py

```diff
@@ -1342,6 +1342,9 @@ def _resolve_parameters_(self,
                 param_resolver))
         return resolved_circuit
 
+    def _qasm_(self) -> str:
+        return self.to_qasm()
+
     def to_qasm(self,
                 header: str = 'Generated from Cirq',
```

Comment:
should this pull in the version of Cirq it was generated from?

Reply:
Done.